### DISCUSSION TITLE
Introductions! 👋

### DISCUSSION THREAD

**Discussion by edakturk14** [BuidlGuidl/batch1.buidlguidl.com#1] [General]
Hey everyone! 

It's Eda here, I've been part of the BuidlGuidl for almost 1.5 years! I'm a developer and I spend a lot of time [writing](https://mirror.xyz/edatweets.eth) about different topics and concepts in crypto. I enjoy working on developer education and creating content/documentation.

I'm very excited for batch#1, hope that it will be a very fun project. If you have any questions, feel free to ask anytime!

**Comment by GusMeg** [comment]
Hi everyone 👋 
I'm Gustavo from Montevideo, Uruguay 🇺🇾 .
After finishing my M.Sc. in Chemical Engineering last year, I started studying about blockchain technology in my spare time and I was fascinated. In the following months I kept on attending seminars, bootcamps and... here I am, starting my Web 3 developer journey 🚀 .
Huge thanks to Eda, Austin and the rest of the team for organizing this whole experience!!! I'm eager to learn and get to know everybody.

**Reply by swellander** [reply]
Nice to meet you!

**Comment by swellander** [comment]
Hello all, my name is Sam (websurfer.eth)! :wave: :ocean:  

Checking in from (the currently very chilly :cold_face:) Bellingham, Washington in the US :us: I've worked as a web2 developer for the last 4 years, and part time for [a web3 startup](https://app.patchwallet.com/) over the last year. I've been curious about this community for a little while now and am excited to have finally made the time to dive in!

Nice to meet you, Eda and Gustavo, and I'm excited to meet the rest of the batch soon!

**Comment by mrpapawheelie** [comment]
🚀Hi All, I'm Papa Wheelie, from Arizona. 

I'm excited to be here with you in Batch #1! 

I was originally introduced to Scaffold-ETH with a Mars Shot Bot NFT a few years ago, which sparked my interest in the project. I've been dabbling in blockchain since 2013, and forking repos here and there for the past few years mainly for fun and to learn. I have been a DevOps manager for the past decade, I've always enjoyed solving issues, and strategic thinking, which led me to the first few challenges of SRE and essentially into this batch. Looking forward to learning and contributing.

**Comment by Yassirakh** [comment]
Hi everyone, I'm Yassir from Morocco, currently living in France ( yassirakh.eth).

I've been working as a web2 dev mainly on PHP and symfony tech, and also been doing some data analytics. Very excited to start learning and building cool stuff on EVMs.

**Comment by PrintRH** [comment]
Hellooooooo, BG Fams 👋 !!

RH here! Originally from Malaysia 🇲🇾  but have currently travelled to the other side of the world, US 🇺🇸 !
Have had various experiences throughout my academics and career; from anti-cancer research, systems engineering, integrations specialist and many more. Despite these experiences, I still find Ethereum and web3 fueling my passion deeply. Looking forward to the day when I can contribute and devote all my time into web3.

In the meantime, I am super excited to learn and collaborate with all of you here! Let's BUIDL 🏗️ 

![You can tell a lot about me from this gif](https://media1.tenor.com/m/uWOd1s8XbDAAAAAC/oh-hello-there.gif)

**Comment by syed-ghufran-hassan** [comment]
Hello everyone I am ghufran from Pakistan,
 
I am electrical engineer by profession and working as project manager in the IT firm affiliated with UK. I am keen towards open source development to contribute towards the community and learn from my peers.

**Comment by kaviraj-j** [comment]
Hello all, this is Kaviraj from India.
I'm a Computer Science student. I've been building web applications for almost 2 years and started learning about Blockchain and Web3 apps due to my interest in this space. I'm looking forward to collaborating with you all :) Also thanks to Eda, Carlos, Austin, and the rest of the BG team!

**Comment by carletex** [comment]
Hey all and welcome to BG!! 

My name is Carlos, I've been part of the BuidlGuidl since 2021. I've been a software developer for more than 10 years (mainly PHP & JS) and went _onchain_ a couple of years ago.

During this time I've built things like the [SpeedRunEthereum](https://speedrunethereum.com/) platform, the [BuidlGuidl App](https://app.buidlguidl.com/), [ABI Ninja](https://abi.ninja/) & [Scaffold-ETH 2](https://github.com/scaffold-eth/scaffold-eth-2). Inside the BuidlGuild, I also run the [Sand Garden](https://sandgarden.buidlguidl.com/).

I love learning and building things 👾

Alongside other BG members, I'll be helping you all to go through this batch. Hope you all have fun!

**Comment by KannuSingh** [comment]
Hello everyone,

I'm Karandeep and thrilled to join this community. Over the past 2 years, I've been immersed in learning the web3 stack. I have a passion for acquiring new knowledge and creating consumer-focused solutions. I'm eager to collaborate with all of you!

**Comment by JollyRogerz** [comment]
Hey Peepz, 
I'm JollyV from Portugal, been around blockchain since the 2017 cycle, since 2020 I went full-in on crypto and blockchain, researching projects to invest in, trading and coding some stuff, developed some pwa dapps for the arweave permachain, a personal trading indicator for TV with pinescript, and did some youtube tutorials on token and nft minting, also did a gamejam for the Sandbox.
My background is in Game Design with a master thesis titled (Gaming and blockchain, the game to solve social isolation). After college i worked as a fullstack dev for almost a year and a half, this was pre covid from late 2018 to early 2020.
I'm happy for finnaly getting my hands on learning solidity with these challenges and other peers :)
